tests/brew/test_create_dirs.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import annotations
```

Comment:
i don't think this entire file is useful - integration tests here provide more than sufficient coverage because if dirs aren't right then 

that said, since we don't run integration mac arm, i'd add an assert that mock_create_dirs was called with the expected /opt/ thing in test_install_brew_not_installed_non_intel_mac

Reply:
True, but unit tests can be run locally and are much faster so if a change does break devenv, it is better to know sooner